packages/react-dom/package.json

```diff
@@ -29,6 +29,7 @@
     "server.js",
     "server.browser.js",
     "server.node.js",
+    "fizz.js",
```

Comment:
hm. Why doesn't this break `ReactDOMFizzServer-test.js` in build tests? I'll need to look into that.

Reply:
Because npm entry point (see npm folder) directly points the bundle. The entry point above (which you redirect) only used by Jest. So technically this change isn’t even necessary right now. But would be less confusing.

Reply:
For consistency you can make `npm/fizz` also do a re export. Like `npm/server` does.